requirements.txt

```diff
@@ -1,50 +1,71 @@
 asgiref==3.7.2
+attrs==23.2.0
+autobahn==23.6.2
+Automat==22.10.0
+awsebcli==3.20.10
```

Comment:
You can remove this package as well. Since the deployments are handled by the pipeline, we won't need the awsebcli package any more